docs/guide/installation-procedure.md

```diff
@@ -267,10 +267,25 @@ You can pass this warning and install Webots by clicking on the "More info" link
 
 #### From the Installation File
 
-1. Download the `webots-{{ webots.version.package }}.dmg` installation file from our [website](https://cyberbotics.com).
-2. Double click on this file.
-This will mount on the desktop a volume named "Webots" containing the "Webots" folder.
-3. Move this folder to your "/Applications" folder or wherever you would like to install Webots.
+It is better to download Webots using `curl` so that it doesn't get tagged as "downloaded from the Internet" and won't be blocked by macOS Gatekeeper.
+To proceed, open a Terminal app and type the following:
+
+```bash
+curl -L -O https://github.com/cyberbotics/webots/releases/download/{{ webots.version.package }}/webots-{{ webots.version.package }}.dmg
+open webots-{{ webots.version.package }}.dmg
+```
+
+To install Webots only for the current user, without administrator privileges, proceed with:
+```bash
+mkdir ~/Applications
+cp -r /Volumes/Webots/Webots.app ~/Applications
+open ~/Applications/Webots.app    # to launch Webots using the open command
+~/Applications/Webots.app/webots  # to launch Webots directly
+```
+
+Alternatively, you can double-click on the Webots icon to launch it.
```

Comment:
```suggestion
Finally, you can launch Webots typing any of these instructions:
    ```bash  
    open ~/Applications/Webots.app    # to launch Webots using the open command
    ~/Applications/Webots.app/webots  # to launch Webots directly
    ```
Alternatively, you can double-click on the Webots icon to launch it.
```

Indentation of the bash block is just to display it correctly as suggestion